staging/src/k8s.io/client-go/rest/with_retry.go

```diff
@@ -209,14 +211,19 @@ func (r *withRetry) IsNextRetry(ctx context.Context, restReq *Request, httpReq *
 // - we need to seek to the beginning of the request body before we
 //   initiate the next retry, the function should return an error if
 //   it fails to do so.
-func (r *withRetry) prepareForNextRetry(ctx context.Context, request *Request) error {
+func (r *withRetry) prepareForNextRetry(ctx context.Context, request *Request, closer *ResponseCloser) error {
 	if ctx.Err() != nil {
 		return ctx.Err()
 	}
 
 	// Ensure the response body is fully read and closed before
 	// we reconnect, so that we reuse the same TCP connection.
```

Comment:
nit: delete the comment from here, since we moved the comment into the `if` block